debezium-core/src/main/java/io/debezium/relational/TableSchemaBuilder.java

```diff
@@ -389,4 +389,34 @@ protected void addField(SchemaBuilder builder, Column column, ColumnMapper mappe
     protected ValueConverter createValueConverterFor(Column column, Field fieldDefn) {
         return valueConverterProvider.converter(column, fieldDefn);
     }
+
+    /**
+     * Sanitize column names that are illegal in Avro
+     * Must conform to https://avro.apache.org/docs/1.7.7/spec.html#Names
+     *  Legal characters are [a-zA-Z_] for the first character and [a-zA-Z0-9_] thereafter.
+     *
+     * @param column the column object containing the name to be sanitized
+     *
+     * @return the sanitized name.
+     */
+    protected String santizeColumnName(Column column) {
```

Comment:
Could this logic be consolidated with `TopicSelector`. In the light of the new usage, this class probably should be renamed into `NameSelector` or similar.